File: backend/ModelService/Model/MastersModel/LookupTypeModel.cs
using ModelService.CommonModel;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace ModelService.Model.MastersModel
{
    public class LookupTypeModel
    {
        public int Id { get; set; }
        [Required]
        public string? Title { get; set; }
        public string? TitleHindi { get; set; }
        public string? Description { get; set; }
        public string? DescriptionHindi { get; set; }
        [Required]
        public string? Slug { get; set; }
        [JsonIgnore]
        public int UserId { get; set; }
    }
    public class LookupTypeViewModel
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public string? TitleHindi { get; set; }
        public string? Description { get; set; }
        public string? DescriptionHindi { get; set; }
        public string? Slug { get; set; }
        public bool IsActive { get; set; }
        public bool IsDelete { get; set; }
    }
    public class LookupTypeViewListModel
    {
        public int Id { get; set; }
        public string? Title { get; set; }
        public string? TitleHindi { get; set; }
        public string? Description { get; set; }
        public string? DescriptionHindi { get; set; }
        public string? Slug { get; set; }
        public string? ModifiedByName { get; set; }
        public DateTime ModifiedDate { get; set; }
        public bool IsActive { get; set; }
        [JsonIgnore]
        public int TotalRows { get; set; }
    }
    public class LookupTypeFilterModel : IndexModel
    {
        public string? Title { get; set; }
        public string? TitleHindi { get; set; }
        public int IsActive { get; set; }
        public string? FromDate { get; set; }
        public string? ToDate { get; set; }
    }
}

File: admin/src/app/core/models/master-models/LookupTypeModel.ts
import { indexModel } from '../core.model';

export class LookupTypeModel {
    id: number;
    title: string | null;
    titleHindi: string | null;
    description: string | null;
    descriptionHindi: string | null;
    slug: string | null;
}

export class LookupTypeViewModel {
    id: number;
    title: string | null;
    titleHindi: string | null;
    description: string | null;
    descriptionHindi: string | null;
    slug: string | null;
}

export class LookupTypeViewListModel {
    id: number;
    title: string | null;
    titleHindi: string | null;
    description: string | null;
    descriptionHindi: string | null;
    slug: string | null;
    modifiedByName: string | null;
    modifiedDate: string;
    isActive: boolean;
}

export class LookupTypeFilterModel extends indexModel {
    title: string | null;
    titleHindi: string | null;
    fromDate: string | null;
    toDate: string | null;
}